mdata/reorder_buffer.go

```diff
@@ -0,0 +1,83 @@
+package mdata
+
+import (
+	"gopkg.in/raintank/schema.v1"
+)
+
+/*
+ * The reorder buffer keeps a window of data during which it is ok to send data out of order.
+ * Once the reorder window has passed Add() will return the old data and delete it from the buffer.
+ * The reorder buffer itself is not thread safe because it is only used by AggMetric,
+ * which is thread safe, so there is no locking in the buffer.
+ */
+
+type ReorderBuffer struct {
```

Comment:
this doesn't parse right.
see for yourself by running this in the MT dir:
```
go get golang.org/x/tools/cmd/godoc
godoc -http=":6060"
```
and go to http://localhost:6060/pkg/github.com/raintank/metrictank/mdata/#ReorderBuffer

it should be:
```
// ReorderBuffer keeps a window of data during which it is ok to send data out of order.
// Once the reorder window has passed Add() will return the old data and delete it from the buffer.
// The reorder buffer itself is not thread safe because it is only used by AggMetric,
// which is thread safe, so there is no locking in the buffer.
type ReorderBuffer struct {
```